src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java

```diff
@@ -26,15 +37,20 @@ public ChooseCommand()
     @Override
     public void executeCommand(@NotNull SlashCommandEvent event, @Nullable Member sender, @NotNull SlashCommandContext ctx)
     {
-        String firstOption = event.getOption("1st").getAsString();
-        String secondOption = event.getOption("2nd").getAsString();
-        int i = ThreadLocalRandom.current().nextInt(2);
-        if (i == 0)
-        {
-            ctx.reply("I choose... " + firstOption).queue();
-        } else
-        {
-            ctx.reply("I choose... " + secondOption).queue();
+        List<OptionMapping> chooseOptions = event.getOptions();
+        if(checkOptions(chooseOptions)){
```

Comment:
This isn't necessary. The `SlashCommandContext` methods are configured to not allow those mentions